### ISSUE TITLE
Reporting an error in bulkimportusers triggers an error

### ISSUE THREAD

**Issue by rtibbles** [learningequality/kolibri#9554]
## Observed behavior
* When attempting to upload an incorrect filetype to bulk import users, if the file is not the correct format, then it will attempt to added an error message to report back on the task.
* When it does this, something in the reporting has a non-JSON serializable object associated with it `proxy` - which causes an error when attempting to save the serialized task to the database.

## Errors and logs
```
Traceback (most recent call last):
File “/usr/lib/python3/dist-packages/kolibri/core/tasks/job.py”, line 174, in to_json
string_result = json.dumps(working_dictionary)
File “/usr/lib/python3.8/json/init.py”, line 231, in dumps
return _default_encoder.encode(obj)
File “/usr/lib/python3.8/json/encoder.py”, line 199, in encode
chunks = self.iterencode(o, _one_shot=True)
File “/usr/lib/python3.8/json/encoder.py”, line 257, in iterencode
return _iterencode(o, 0)
File “/usr/lib/python3.8/json/encoder.py”, line 179, in default
raise TypeError(f’Object of type {o.class.name} ’
TypeError: Object of type proxy is not JSON serializable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File “/usr/lib/python3/dist-packages/kolibri/core/tasks/worker.py”, line 61, in handle_finished_future
result = future.result()
File “/usr/lib/python3.8/concurrent/futures/_base.py”, line 437, in result
return self.__get_result()
File “/usr/lib/python3.8/concurrent/futures/_base.py”, line 389, in __get_result
raise self._exception
File “/usr/lib/python3.8/concurrent/futures/thread.py”, line 57, in run
result = self.fn(*self.args, **self.kwargs)
File “/usr/lib/python3/dist-packages/kolibri/core/tasks/job.py”, line 117, in execute_job
result = func(*args, **kwargs)
File “/usr/lib/python3/dist-packages/kolibri/dist/django/core/management/init.py”, line 131, in call_command
return command.execute(*args, **defaults)
File “/usr/lib/python3/dist-packages/kolibri/dist/django/core/management/base.py”, line 330, in execute
output = self.handle(*args, **options)
File “/usr/lib/python3/dist-packages/kolibri/core/tasks/management/commands/base.py”, line 119, in handle
return self.handle_async(*args, **options)
File “/usr/lib/python3/dist-packages/kolibri/core/auth/management/commands/bulkimportusers.py”, line 852, in handle_async
self.exit_if_error()
File “/usr/lib/python3/dist-packages/kolibri/core/auth/management/commands/bulkimportusers.py”, line 769, in exit_if_error
self.job.save_meta()
File “/usr/lib/python3/dist-packages/kolibri/core/tasks/job.py”, line 244, in save_meta
self.storage.save_job_meta(self)
File “/usr/lib/python3/dist-packages/kolibri/core/tasks/storage.py”, line 287, in save_job_meta
self._update_job(job.job_id, extra_metadata=job.extra_metadata)
File “/usr/lib/python3/dist-packages/kolibri/core/tasks/storage.py”, line 300, in _update_job
orm_job.saved_job = job.to_json()
File “/usr/lib/python3/dist-packages/kolibri/core/tasks/job.py”, line 178, in to_json
raise TypeError(
TypeError: Job objects need to be JSON-serializable: Object of type proxy is not JSON serializable
```

## Expected behavior
Error should be properly reported to the end user, not trigger an unhandled error message.

## User-facing consequences
Confusion!

## Steps to reproduce
Upload an invalid file for CSV import

## Context
https://community.learningequality.org/t/error-importing-users/2491

**Comment by jredrejo** [comment]
The problem is not the format of the file but the `gettext_lazy` and `pgettext_lazy` functions used to translate the errors that were not being converting the msg to `str` before being serialized. Not able to reproduce the problem, but I've added more guards to force the conversion and ensure there're not proxies in the serialized `job`.

**Comment by rtibbles** [comment]
Fixed in #9557
